examples/AlwaysMints.purs

```diff
@@ -0,0 +1,146 @@
+-- | This module demonstrates how the `Contract` interface can be used to build,
+-- | balance, and submit a smart-contract transaction. It creates a transaction
+-- | that mints a value using the `AlwaysSucceeds` policy
+-- |
+-- | * Prerequisites
+-- |   - A Chromium-based browser (for Nami compatibility)
+--
+-- |   - A Nami wallet funded with test Ada ("tAda") and collateral set, If you need
+-- |     tAda, visit https://testnets.cardano.org/en/testnets/cardano/tools/faucet/
+--
+-- | * How to run
+--
+-- |   The `Contract` interface requires several external services to be running.
+-- |   From the repository root, run `nix run .#ctl-runtime` to launch all
+-- |   required services
+--
+-- |   Once all of the services are *fully synced*, run:
+--
+-- |   - `make run-dev` and visit `localhost:4008`. You may be prompted to enable
+-- |     access to your wallet if you have not run this example before. You will
+-- |     also be prompted to sign the transaction using your Nami password
+
+module Examples.AlwaysMints (main) where
+
+import Contract.Prelude
+
+import Contract.Monad
+  ( ContractConfig(ContractConfig)
+  , launchAff_
+  , liftContractM
+  , liftedE
+  , liftedM
+  , logInfo'
+  , runContract_
+  , traceContractConfig
+  )
+import Contract.Prim.ByteArray (byteArrayFromAscii)
+import Contract.ScriptLookups as Lookups
+import Contract.Scripts (MintingPolicy)
+import Contract.Transaction
+  ( BalancedSignedTransaction(BalancedSignedTransaction)
+  , balanceAndSignTx
+  , submit
+  )
+import Contract.TxConstraints as Constraints
+import Contract.Value as Value
+import Contract.Wallet (mkNamiWalletAff)
+import Data.Argonaut (decodeJson, fromString)
+import Data.BigInt as BigInt
+
+main :: Effect Unit
+main = launchAff_ $ do
+  wallet <- Just <$> mkNamiWalletAff
+  cfg <- over ContractConfig _ { wallet = wallet } <$> traceContractConfig
+  runContract_ cfg $ do
+    mp <- liftContractM "Invalid script JSON" $ alwaysMintsPolicy
+    cs <- liftedM "Cannot get cs" $ Value.scriptCurrencySymbol mp
+    tn <- liftContractM "Cannot make token name"
+      $ Value.mkTokenName
+      =<< byteArrayFromAscii "TheToken"
+
+    let
+      -- Note that CTL does not have explicit equivalents of Plutus'
+      -- `mustPayToTheScript` or `mustPayToOtherScript`, as we have no notion
+      -- of a "current" script. Thus, we have the single constraint
+      -- `mustPayToScript`, and all scripts must be explicitly provided to build
+      -- the transaction (see the value for `lookups` below as well)
```

Comment:
Is this an old comment?

Reply:
Hmmm. That's odd, I removed that in my most recent commit